### DISCUSSION TITLE
在使用CLOUMN租户模式后，如果一张表某些业务需要使用租户方式，其他业务不需要租户方式如何处理？

### DISCUSSION THREAD

**Discussion by zbh0523** [zuihou/zuihou-admin-cloud#169] [02.提问] [answered]
@Override
    protected List<InnerInterceptor> getPaginationBeforeInnerInterceptor() {
        List<InnerInterceptor> list = super.getPaginationBeforeInnerInterceptor();
        Boolean isDataScope = databaseProperties.getIsDataScope();
        if (isDataScope) {
            list.add(getDataScopeInnerInterceptor());
        }
        return list;
    }

已在MyBatis配置类中修改为上述代码，List<InnerInterceptor> list = super.getPaginationBeforeInnerInterceptor();，经测试实现了租户模式，所有CRUD自动注入tenant_id，但是其他业务接口也用到这张表，但是不想使用租户方式，比如查所有的数据等，应如何处理？

**Comment by zuihou** [comment, marked as answer]
在 mapper 上加注解： @InterceptorIgnore
